components/Visit/VisitInfoForm.tsx

```diff
@@ -134,17 +157,10 @@ export default function VisitInfoForm({
                     <FormItem
                         type="checkbox"
                         id="clothingBoy"
-                        label="Kids (Boy)"
+                        label="Kids"
                         checked={!!visitData.clothingBoy}
                         onChange={handleChange('clothingBoy')}
                     />
-                    <FormItem
-                        type="checkbox"
-                        id="clothingGirl"
-                        label="Kids (Girl)"
-                        checked={!!visitData.clothingGirl}
-                        onChange={handleChange('clothingGirl')}
-                    />
```

Comment:
please handle edge cases when merging Boy and Girl.
What if existing record has girl checked, but not boy and vise versa. Is the KidQ input box displayed when existing record has girl checked only?